app/views/registrations/_contact_info.html.haml

```diff
@@ -2,10 +2,19 @@
   .field
     = f.label :phone
     = f.text_field :phone
+  .field
+    = f.label :receive_sms, 'Text Notifications'
+    %div
+      %label
+        = f.radio_button :receive_sms, true, checked: (params[:receive_sms] == nil ? true : params[:receive_sms])
+        Send Me Text Message Notifications
+      %label
+        = f.radio_button :receive_sms, false
+        Don't Send Me Any Text Messages
   .field
     = f.label :local_phone
     = f.text_field :local_phone
-    %span.smalltext (phone number during the U.S. Go Congress)
+    %span.smalltext (A U.S. phone number is required to receive notifications during the Congress. Enter 10 digits with no spaces or dashes.)
```

Comment:
We could also state that message and data rates may apply.

Reply:
Good call.